plugins/inputs/sqlserver/azuresqlqueries.go

```diff
@@ -382,6 +382,116 @@ HAVING
+	rs.avg_physical_io_reads as avg_physical_io_reads,
+	rs.max_logical_io_writes as max_logical_io_writes,
+	rs.avg_logical_io_writes as avg_logical_io_writes,
+	rs.execution_type,
+	rs.count_executions as count_executions,
+	rs.max_cpu_time as max_cpu_time,
+	rs.avg_cpu_time as avg_cpu_time,
+	rs.max_dop as max_dop,
+	rs.avg_dop as avg_dop,
+	rs.max_rowcount as max_rowcount,
+	rs.avg_rowcount as avg_rowcount,
+	rs.max_query_max_used_memory AS max_query_max_used_memory,
+	rs.avg_query_max_used_memory AS avg_query_max_used_memory,
+	rs.max_duration as [max_duration],
+	rs.avg_duration as avg_duration,
+	rs.max_num_physical_io_reads as max_num_physical_io_reads,
+	rs.avg_num_physical_io_reads as avg_num_physical_io_reads,
+	rs.max_log_bytes_used as [max_log_bytes_used],
+	rs.avg_log_bytes_used as avg_log_bytes_used
+FROM sys.query_store_query q WITH (NOLOCK)
+	JOIN sys.query_store_plan p WITH (NOLOCK) ON q.query_id = p.query_id
+	JOIN sys.query_store_runtime_stats rs WITH (NOLOCK) ON p.plan_id = rs.plan_id
+	JOIN sys.query_store_runtime_stats_interval rsi WITH (NOLOCK) ON rs.runtime_stats_interval_id = rsi.runtime_stats_interval_id
+WHERE rsi.start_time >= @currIntervalStartDate AND rsi.end_time <= @currIntervalEndDate;
+
+SELECT CAST(@currIntervalEndTimestamp AS nvarchar(20)) AS QueryData;
+END;
+`
+
+const sqlAzureDBQueryStoreWaitStatistics = sqlAzureDBPartQueryPeriod + `
+SET DEADLOCK_PRIORITY -10;
+IF SERVERPROPERTY('EngineEdition') = 5  -- Is this Azure SQL DB?
+BEGIN
+SELECT
+	'sqlserver_azuredb_querystore_wait_stats' AS [measurement],
+	CONVERT(varchar(36), NEWID()) AS [measurement_id],
+	REPLACE(@@SERVERNAME, '\', ':') AS [sql_instance],
+	DB_NAME() AS [database_name],
+	CONVERT(datetime, rsi.start_time, 1) AS interval_start_time,
+	CONVERT(datetime, rsi.end_time, 1) AS interval_end_time,
+	CONVERT(BIGINT, DATEDIFF(day, 0, CONVERT(datetime, rsi.start_time, 1))) * @magicNumberDays + DATEDIFF(second, 0, DATEADD(day,-DATEDIFF(day, 0, CONVERT(datetime, rsi.start_time, 1)), CONVERT(datetime, rsi.start_time, 1))) * @magicNumberSeconds AS interval_start_time_d,
+	CONVERT(BIGINT, DATEDIFF(day, 0, CONVERT(datetime, rsi.end_time, 1))) * @magicNumberDays + DATEDIFF(second, 0, DATEADD(day,-DATEDIFF(day, 0, CONVERT(datetime, rsi.end_time, 1)), CONVERT(datetime, rsi.end_time, 1))) * @magicNumberSeconds AS interval_end_time_d,
+	q.query_id,
+	CONVERT(nvarchar(max), q.query_hash, 1) as query_hash,
+	p.plan_id,
+	CONVERT(nvarchar(max), q.last_compile_batch_sql_handle, 1) as last_compile_batch_sql_handle,
+	CONVERT(nvarchar(max), p.query_plan_hash, 1) as query_plan_hash,
+	ws.wait_category_desc AS wait_category_s,
+	ws.execution_type AS exec_type_d,
+	ws.total_query_wait_time_ms AS total_query_wait_time_ms_d,
+	ws.max_query_wait_time_ms AS max_query_wait_time_ms_d,
+	q.query_parameterization_type AS query_param_type_d
+FROM sys.query_store_query q WITH (NOLOCK)
+	JOIN sys.query_store_plan p WITH (NOLOCK) ON q.query_id = p.query_id
```

Comment:
Do we need this dmv? only thing being returned is plan_id which I beleive is in runtimestats